hudi-client/hudi-client-common/src/main/java/org/apache/hudi/metrics/Metrics.java

```diff
@@ -118,6 +150,11 @@ public static void registerGauges(Map<String, Long> metricsMap, Option<String> p
     metricsMap.forEach((k, v) -> registerGauge(metricPrefix + k, v));
   }
 
+  public static void registerGauges(Map<String, Long> metricsMap, Option<String> prefix, Map<String, String> labels) {
```

Comment:
There are two `registerGauges` and this one is not getting used anywhere other than the tests. Exposing a public method just for tests does is not a good practice. Is there a better way to refactor?

Reply:
Removed it.